roles/openshift_aws/tasks/provision.yml

```diff
@@ -7,6 +7,43 @@
   name: create s3 bucket for registry
   include: s3.yml
 
+- when: openshift_aws_create_security_groups
```

Comment:
We should group these together.

Reply:
Grouped in latest with block/when statement.